guide/popular-topics/common-questions.md

```diff
@@ -377,3 +377,52 @@ Assuming the process is to be done for the guild the message is sent in.
 ```
 
 </branch>
+
+### How do I check the ping of my bot?
+
+There are several ways to get the **"ping"** of the bot. Two of the ways can be by the **WebSocket ping** or by the **message latency**.
+
+<branch version="11.x">
+
+<!-- eslint-skip -->
+
+The first example gets the websocket ping. The `ping` property can be found on the Client object.
+
+```js
+// The Client is where you can get the ping property.
+message.channel.send(`My heartbeat ping is ${client.ping}ms!`);
```

Comment:
This section the guide uses `<message>` and `<client>` as detailed in the legend on top of this page, the comment is not very informative, can be left out entirely